src/Core/src/Platform/HandlerNotFoundException.cs

```diff
@@ -11,6 +11,12 @@ public HandlerNotFoundException()
 		{
 		}
 
+		public HandlerNotFoundException(IElement element) :
+			this($"Handler not found for view {element}.")
```

Comment:
This will throw at runtime, but we need to start giving translation to these message.